src/main/java/racingcar/domain/RacingCarGame.java

```diff
@@ -0,0 +1,64 @@
+package racingcar.domain;
+
+import java.util.Objects;
+import java.util.regex.Pattern;
+import racingcar.ErrorMessage;
+
+public class RacingCarGame {
+
+    private static final int MIN_ROUND = 1;
+
+    private static final String NUMBER_PATTERN = "^[0-9]*$";
+    private static final String BLANK = " ";
+
+    private final Cars cars;
+    private final int totalRound;
+    private final RoundHistory roundHistory;
+
+    private RacingCarGame(Cars cars, int totalRound, RoundHistory roundHistory) {
+        this.cars = cars;
+        this.totalRound = totalRound;
+        this.roundHistory = roundHistory;
+    }
+
+    public static RacingCarGame create(Cars cars, String totalRound, RoundHistory roundHistory) {
+        return new RacingCarGame(cars, validateTotalRound(totalRound), roundHistory);
+    }
+
+    private static int validateTotalRound(String totalRound) {
+        if (isInvalidTotalRound(totalRound)) {
+            throw new IllegalArgumentException(ErrorMessage.INVALID_TOTAL_ROUND);
```

Comment:
정적 임포트를 통해 가독성을 증진시킬 수 있겠네요 :)
```suggestion
            throw new IllegalArgumentException(INVALID_TOTAL_ROUND);
```